src/main/java/seedu/duke/command/EditEntryCommand.java

```diff
@@ -67,7 +68,6 @@ public EditEntryCommand(int index, double amount, String description, String dat
             throw new IllegalArgumentException("Invalid date format: " + date, e);
```

Comment:
Does FinanceBuddyException alreaady handle the exception properly?

Reply:
Edited code to handle appropriately